src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Properties/launchSettings.json

```diff
@@ -12,7 +12,7 @@
     "IIS Express": {
       "commandName": "IISExpress",
       "launchBrowser": true,
-      "launchUrl": "api/values",
+      "launchUrl": "api/v1/actors",
```

Comment:
remember to also update the values in studioOps project with this endpoint

Reply:
updated